config/templates/stack-template.json

```diff
@@ -1105,19 +1097,19 @@
       },
       "Type": "AWS::EC2::Subnet"
     }
+    {{end}}
     {{if $.ElasticFileSystemID}}
     ,
-    "{{$subnetLogicalName}}MountTarget": {
+    "{{$subnet.SubnetLogicalName}}MountTarget": {
```

Comment:
Note: When the `$subnet` is an existing one, we probably need to create a mount target if and only if the mount target doesn't exist in for the pair of the subnet and the EFS.
See https://github.com/coreos/kube-aws/issues/208#issuecomment-272036205